django/applications/catmaid/control/similarity.py

```diff
@@ -460,7 +462,7 @@ def add_delayed(self, project_id, user_id, name, matching_skeleton_ids,
                     probability=probability)
 
         config = NblastConfig.objects.create(project_id=project_id,
-            user=user, name=name, status='queued',
+            user=user, name=name, status='queued', # FIXME: user is not defined
```

Comment:
`user` should be `user_id`, which is passed in as an argument. This function isn't used presently.